x-pack/plugins/security_solution/public/explore/components/risk_score/risk_score_onboarding/risk_score_doc_link.tsx

```diff
@@ -8,12 +8,22 @@
 import { EuiLink } from '@elastic/eui';
 import React, { useMemo } from 'react';
 import { RiskScoreEntity } from '../../../../../common/search_strategy';
-import {
-  RISKY_HOSTS_DOC_LINK,
-  RISKY_USERS_DOC_LINK,
-  RISKY_ENTITY_SCORE_DOC_LINK,
-} from '../../../../../common/constants';
 import { LEARN_MORE } from '../../../../overview/components/entity_analytics/risk_score/translations';
+import { useKibana } from '../../../../common/lib/kibana';
+
+const useLearnMoreLinkForEntity = (riskScoreEntity?: RiskScoreEntity) => {
```

Comment:
Not necessarily a suggestion, more of a question:

Do you think there would be value in adding a test case for this, now that we're using more dynamic links instead of them just being hardcoded? I assume it would involve mocking the docLinks service, so not sure how much value we'd get, but thought I'd ask!

Reply:
I did debate testing it, but I concluded the tests would be too tied to the implementation to give much value.

Reply:
Makes sense to me!